packages/mux-player/src/media-theme-mux.html

```diff
@@ -58,16 +58,15 @@
       cursor: not-allowed;
     }
 
-    :host(:not([audio])) media-captions-menu-button::part(listbox) {
```

Comment:
I didn't actually try to see if I can add a part to the listbox, but since in this implementation it's in the lightdom, we don't need it. Likely to change again in the next selectmenu update.